torch/serialization.py

```diff
@@ -406,28 +407,74 @@ def persistent_id(obj: Any) -> Optional[Tuple]:
                               "for correctness upon loading.")
             return ('module', obj, source_file, source)
 
-        elif torch.is_storage(obj):
+        if isinstance(obj, torch.storage.TypedStorage) or torch.is_storage(obj):
```

Comment:
Something that will be helpful is to add notes about when code can be deleted (and how to do the deletion), so people don't perpetually tiptoe around TypedStorage

Reply:
I've added some TODOs where we should no longer need to use TypedStorage when we decide to break FC